wolfssl/wolfcrypt/aes.h

```diff
@@ -250,6 +250,21 @@ struct Aes {
     silabs_aes_t ctx;
 #endif
     void*  heap; /* memory hint to use */
+#ifdef WOLFSSL_AESGCM_STREAM
+#if !defined(WOLFSSL_SMALL_STACK) || defined(WOLFSSL_AESNI)
+    ALIGN16 byte streamData[5 * AES_BLOCK_SIZE];
+#else
+    byte*        streamData;
```

Comment:
Is this free'd during Final in SMALL_STACK?  I don't see where.

Reply:
No freed in wc_AesFree().
Didn't want to allocate and free this every time.
Just saving the memory from being allocated for non AES-GCM streaming operations.
wc_AesInit() must be called.